cocos/spine/lib/spine-define.ts

```diff
@@ -1325,8 +1325,11 @@ function overrideProperty_Skin (): void {
         value (slotIndex: number, attachments: Array<spine.SkinEntry>) {
             const vectors = originGetAttachmentsForSlot.call(this, slotIndex);
             const count = vectors.size();
-            attachments = resizeArray(attachments, count);
-            for (let i = 0; i < count; i++) attachments[i] = vectors.get(i);
+            attachments.length = 0;
```

Comment:
assign lenth with count, then set value of array with index is more memory friendly

Reply:
Thanks for your review, it's been fixed.